docs/framework/performance/reliability-best-practices.md

```diff
@@ -236,7 +236,7 @@ For SQL Server, all methods used to introduce synchronization or threading must
 
 ### Do Not Block Indefinitely in Unmanaged Code
 
-Blocking in unmanaged code instead of in managed code can cause a denial of service attack because the CLR is not able to abort the thread.  A blocked thread prevents the CLR from unloading the <xref:System.AppDomain>, at least without doing some extremely unsafe operations.  Blocking using a Win32 synchronization primitive is a clear example of something we cannot allow.  Blocking in a call to `ReadFile` on a socket should be avoided if possible — ideally the Win32 API should provide a mechanism for an operation like this to time out.
+Blocking in unmanaged code instead of in managed code can cause a denial of service attack because the CLR is not able to abort the thread.  A blocked thread prevents the CLR from unloading the <xref:System.AppDomain>, at least without doing some extremely unsafe operations.  Blocking using a Win32 synchronization primitive is a clear example of something we cannot allow.  Blocking in a call to `ReadFile` on a socket should be avoided if possible — ideally the Windows API should provide a mechanism for an operation like this to time out.
```

Comment:
```suggestion
Blocking in unmanaged code instead of in managed code can cause a denial of service attack because the CLR is not able to abort the thread.  A blocked thread prevents the CLR from unloading the <xref:System.AppDomain>, at least without doing some extremely unsafe operations.  Blocking using a Windows synchronization primitive is a clear example of something we cannot allow.  Blocking in a call to `ReadFile` on a socket should be avoided if possible — ideally the Windows API should provide a mechanism for an operation like this to time out.
```